bot/exts/moderation/voice_gate.py

```diff
@@ -144,6 +166,10 @@ async def on_message(self, message: discord.Message) -> None:
 
         # When it's bot sent message, delete it after some time
         if message.author.bot:
+            # Comparing the message with the voice ping constant
+            if message.content.endswith(VOICE_PING[-45:]):
```

Comment:
I don’t quite like this, if we ever change the message and the first 45 characters aren’t the same for any reason, the code will break. Also, a user could spam the channel using the same 45 characters. What I think should be done is not deleting messages in `on_message` if the user is a bot, and instead handle deletion were the message is created, you’d just have to delete the message sent line 147 in the same function.

Reply:
The message is only checked if the message is from a bot, so a user couldn't trigger it

Reply:
> The message is only checked if the message is from a bot, so a user couldn't trigger it

Right, I totally missed that. But I still think there’s a better solution. 😄

Reply:
Corrected in latest commit